### PR TITLE
C/C++ 1.22: finalize change notes

### PR DESCRIPTION
**Pull request by felicity-semmle**
The first commit removes unwanted boilerplate text and sorts the table alphabetically.
The second commit makes some minor text changes.

@Semmle/cpp - I find line 24 difficult to understand and would appreciate a suggestion or direct edit to that line. It looks as if it would be improved with either punctuation, or backticks to indicate code elements.

### CODE REVIEW

File: change-notes/1.22/analysis-cpp.md [hunk 1]
```diff
@@ -29,7 +24,7 @@
 - Fixed the `LocalScopeVariableReachability.qll` library's handling of loops with an entry condition is both always true upon first entry, and where there is more than one control flow path through the loop condition.  This change increases the accuracy of the `LocalScopeVariableReachability.qll` library and queries which depend on it.
```
Comment (nickrolfe) [inline change-notes/1.22/analysis-cpp.md]:
```suggestion
- Fixed the `LocalScopeVariableReachability.qll` library's handling of loops where the entry condition is always true on first entry, and where there is more than one control flow path through the loop condition. This change increases the accuracy of the `LocalScopeVariableReachability.qll` library and queries that depend on it.
```

**Comment by geoffw0** [comment]
LGTM.  We need to be sure that the change notes reflect which (if any) of @jbj's recent changes to the dataflow / taint library we decide to ship with.

File: change-notes/1.22/analysis-cpp.md (hunk 1 above)
Comment (felicity-semmle) [reply change-notes/1.22/analysis-cpp.md]: Thanks

**Comment by geoffw0** [comment]
I've added my change here.  I was expecting it to 'suggest' the change but it's simply added it as a commit, so @felicity-semmle please review that commit and reword as necessary.

**Comment by felicity-semmle** [comment]
Thanks @geoffw0. That looks fine unless we want to make it more prominent. I know that you've all put a lot of effort into improving the library documentation.

**Comment by geoffw0** [comment]
That's true, but given that (1) the work is not complete in this release and (2) users don't need to know about it or do take actions to benefit, I don't think it needs to be especially prominent.

**Pull request merged**